docs/design/feedback.rst

```diff
@@ -0,0 +1,119 @@
+.. vale off
+
+Providing effective user feedback
+=================================
+
+.. vale on
+
+When developing features, it's crucial to ensure that Users receive clear feedback and guidance when certain information or data isn't available. Instead of simply hiding tabs or displaying zeroed metrics, for example, adopt a proactive approach to inform and guide Users.
+
+Fundamental principles
+----------------------
+
+The principles of visibility, transparency, and guidance Form the foundation of an intuitive and informative User experience.
```

Comment:
```suggestion
The principles of visibility, transparency, and guidance form the foundation of an intuitive and informative user experience.
```
No need to capitalise here, we're not referring to a Mautic form or the Mautic user. Use vale off / vale on instead.

Reply:
So still I have to do something?

Reply:
@favour-chibueze can pick up my comments in the review to address the Vale issues she was working through, I think.